code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -869,3 +869,59 @@
 		log_game("DYNAMIC: Not enough players volunteered for the ruleset [name] - [candidates.len] out of [required_candidates].")
 		return FALSE
 	return TRUE
+
+//////////////////////////////////////////////
+//                                          //
+//           NINJA      (GHOST)             //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/midround/from_ghosts/ninja
+	name = "Space Ninja"
+	midround_ruleset_style = MIDROUND_RULESET_STYLE_HEAVY
+	antag_flag = ROLE_NINJA
+	required_type = /mob/dead/observer
+	antag_datum = /datum/antagonist/ninja
+	enemy_roles = list(JOB_NAME_SECURITYOFFICER, JOB_NAME_DETECTIVE, JOB_NAME_WARDEN, JOB_NAME_HEADOFSECURITY, JOB_NAME_CAPTAIN)
+	required_enemies = list(2,2,2,2,2,2,2,2,2,2)
+	required_candidates = 1
+	weight = 3
+	cost = 9
```

Comment:
imo the cost should be higher. maybe 10 or 11

Reply:
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/26465327/ebf67ac7-e744-4d1d-a343-454081d17ae1)

I think its in a good position considering the relative costs of them, only one that I think is off is pirates which I think are more impactful that space ninja however that is more a matter of pirates being cheap than ninja being too expensive

Reply:
Ninja is usually less damaging than pirates, now they have had their ability to perma-capture people removed too. Its a fun antag and most people seem to like it.

Reply:
OK